wordpress-mobile/WordPress-iOS
# Open magic links to correct screen

Fixes magic link signup epilogue getting skipped.

Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/445

### To test:
1. Get started > enter new email address (+ modifiers work) > Continue > Send link by email
2. Check email and activate link
3. **Expected:** Magic link processes and goes to the signup epilogue.

PR submission checklist:
